src/multivariate/mvnormal.jl

```diff
@@ -80,6 +80,27 @@ abstract type AbstractMvNormal <: ContinuousMultivariateDistribution end
 insupport(d::AbstractMvNormal, x::AbstractVector) =
     length(d) == length(x) && all(isfinite, x)
 
+"""
+    minimum(d::AbstractMvNormal)
+
+Return the minimum of the support of `d`.
+"""
+minimum(d::AbstractMvNormal) = fill(-Inf, length(d))
```

Comment:
Is there a type you can pull? this would default to `Float64`

Reply:
also, default container would be an array, do you want this over a tuple?

Reply:
and do what with it? Inf is not parametric.

Reply:
```
julia> convert(Float32, Inf)
Inf32
```

Reply:
Good call